sidebars.js

```diff
@@ -4,92 +4,351 @@ module.exports = {
     {
       type: "category", // doc(default), 'link'(external links), 'ref'(link without sidebar)
       label: "Get Started",
+      collapsed: false,
       items: [
         {
           type: "category",
-          label: "Zowe Fundamentals",
+          label: "Zowe fundamentals",
           items: [
             "getting-started/overview", //folder-name/filename
             "getting-started/zowe-architecture",
             "getting-started/freqaskques",
-            {
-              type: "category",
-              label: "Introductions Parent",
-              items: [
-                "getting-started/doc2",
-                "getting-started/introductions/doc1",
-                {
-                  type: "category",
-                  label: "Introductions Child 1",
-                  items: [
-                    "getting-started/doc4",
-                    {
-                      type: "category",
-                      label: "Introductions Child 2",
-                      items: ["getting-started/doc7", "getting-started/doc8"],
-                    },
-                  ],
-                },
-              ],
-            },
-          ],
-        },
-        // 'getting-started/release-notes', //type: doc, id ='getting-started/release-notes
-        {
-          type: "doc",
-          id: "getting-started/release-notes",
-          customProps: {
-            /* props */
-            title: "Review Notes",
-            sidebar_label: "Overview",
-          },
+          ],
         },
+        "getting-started/summaryofchanges.md",
         {
           type: "category",
           label: "Zowe quick start",
           items: [
-            "getting-started/zowe-getting-started",
-            "getting-started/cli-getting-started",
+            "getting-started/zowe-getting-started.md",
+            "getting-started/cli-getting-started.md",
+          ],
+        },
+        {
+          type: "category",
+          label: "Learning paths",
+          items: [
+            "getting-started/user-roadmap-apiml.md",
+            "getting-started/user-roadmap-app-framework.md",
+            "getting-started/user-roadmap-zowe-cli.md",
+            "getting-started/user-roadmap-zowe-explorer.md",
+            "getting-started/user-roadmap-client-sdk.md",
+            "getting-started/user-roadmap-zowe-mobile.md",
           ],
         },
+        "getting-started/zowe-resources.md",
       ],
     },
   ],
+  user-guide: [
```

Comment:
Folders having more than a single word in their name will be in inverted commas (**""**) like `"user-guide"`.

Reply:
I corrected it with the changes :)